### ISSUE TITLE
Is there a need to restore the dataloader condition

### ISSUE THREAD

**Issue by ryanxingql** [open-mmlab/mmediting#959]
Hi,
I am wondering that is there a need to restore the dataloader condition.
Every time we restore the model, we only restore the model parameters, optimizer, and scheduler.
The dataloader will start from the beginning, resulting in repeated training over the data in front.
To actually restore the training condition, we may need to restore the dataloader condition, i.e., skip the trained iters or epochs of data.
If this function is necessary, I may try to implement it.
